src/classes/groundplaces.ts

```diff
@@ -25,21 +48,95 @@ export class GroundPlaces {
   public autocomplete(query: string, filters: AutoCompleteFilters[]): AutoComplete {}
 
   /**
-   * @description Create the stopGroup with the values given.
+   * @description Create a new StopGroup from a SegmentProviderStop.
+   * @param {SegmentProviderStop} segmentProviderStop - The SegmentProviderStop on which is based the StopGroup.
+   * @param {StopGroupInfos} stopGroupInfos - StopGroup informations.
    * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
-   * @param {StopGroup} stopGroupInfos - StopGroup informations.
-   * @returns {StopGroup}
+   * @returns {StopGroup|Error}
```

Comment:
The Simple the better. 
@returns {StopGroup} 
(and could throw an error but this is not a type)